tls/s2n_internal.h

```diff
@@ -53,3 +50,12 @@ S2N_PRIVATE_API int s2n_connection_get_config(struct s2n_connection *conn, struc
  */
 S2N_PRIVATE_API int s2n_config_add_cert_chain(struct s2n_config *config,
         uint8_t *cert_chain_pem, uint32_t cert_chain_pem_size);
+
+/*
+ * Attempts to flush any data buffered for sending.
+ *
+ * This method is not sufficient to complete a previous partial send. It can only
+ * attempt to flush data that has been encrypted and buffered, not data that
+ * is still waiting for encryption.
+ */
+S2N_PRIVATE_API int s2n_flush(struct s2n_connection *conn, s2n_blocked_status *blocked);
```

Comment:
We could remove it, but we don't have to remove it as long as the two definitions match. I just left it to avoid needing to include internal.h in more file. The downside is that if we ever update the definition, we'll need to either update both places or remove one definition. Thoughts?

Reply:
Makes sense!